src/seismometer/data/pandas_helpers.py

```diff
@@ -543,9 +535,11 @@ def event_value_name(event_value: str) -> str:
     return val
 
 
-def valid_event(dataframe: pd.DataFrame, event: str) -> pd.DataFrame:
-    """Filters a dataframe to valid predictions, where the event value has not set to -1."""
-    return dataframe[dataframe[event_value(event)] >= 0]
+def is_valid_event(dataframe: pd.DataFrame, event: str, ref: str) -> pd.DataFrame:
+    """Creates a mask if events occur after the reference time, defaults to True."""
```

Comment:
If the event is not in the dataframe, shouldn't this all be false so we end up with no rows?

Reply:
I can update doc if you think it'd be useful, but have generally been thinking about timing being  respected as a filter only when it exists.  If it's "unknown", I wouldn't want this to hide it.

I could also update the name to is_not_invalid_event, if you think the default is more intuitive that way.

Reply:
I think just updating the header to something of the effect : "If the event has a timestamp, verify the timestamp is after the prediction"